packages/web-twig/src/Resources/components/Header/README.md

```diff
@@ -46,126 +106,394 @@ With Html syntax lexer (enabled by default):
+It holds the toggle button by default, but you can add as many custom elements
+as the free space in Header allows.
 
 ```twig
-<Header isInverted>
-  <a href="/">
-    <img
-      src="https://www.example.com/logo.png"
-      width="65"
-      height="24"
-      alt="Spirit"
-    />
-  </a>
-  <NavbarToggle />
-  <Navbar variant="bar">
-    <NavbarClose />
-    <NavbarActions color="primary">
-      <Nav>
-        <NavItem>
-          <NavLink href="/" isCurrent>Nabídky práce</NavLink>
-        </NavItem>
-        <NavItem>
-          <NavLink href="/">Brigády</NavLink>
-        </NavItem>
-        <NavItem>
-          <NavLink href="/">Inspirace</NavLink>
-        </NavItem>
-        <NavItem>
-          <NavLink href="/">Historie odpovědí</NavLink>
-        </NavItem>
-        <NavItem>
-          <NavLink href="/">Zaměstnavatelé</NavLink>
-        </NavItem>
-      </Nav>
-    </NavbarActions>
-    <NavbarActions color="secondary">
-      <ButtonLink color="primary" href="/">Přihlásit</ButtonLink>
-      <ButtonLink color="inverted" href="/">Vstup pro firmy</ButtonLink>
-    </NavbarActions>
-  </Navbar>
-</Header>
+<HeaderMobileActions dialogId="my-header-dialog" />
+```
+
+Toggle button is already part of the mobile actions component. It is linked to
+the [Header Dialog](#header-dialog) via the `dialogId` prop.
+
+#### Custom Mobile Actions
+
+You can place any custom content into the mobile actions component:
+
+```twig
+<HeaderMobileActions dialogId="my-header-dialog">
+  <!-- Mobile-only actions -->
+</HeaderMobileActions>
 ```
 
-## Header components
+#### API
 
-The Header itself consists of many components which cannot be used independently.
+| Prop name         | Type     | Default | Required | Description                     |
+| ----------------- | -------- | ------- | -------- | ------------------------------- |
+| `class`           | `string` | `null`  | no       | Custom CSS class                |
+| `dialogId`        | `string` | —       | yes      | ID of the linked HeaderDialog   |
+| `menuToggleLabel` | `string` | `Menu`  | no       | Label of the menu toggle button |
 
-### Header
+On top of the API options, you can add `data-*` or `aria-*` attributes to
+further extend component's descriptiveness and accessibility. These attributes
+will be passed to the topmost HTML element of the component.
 
-#### Available props
+### Desktop-Only Actions
 
-| Name         | Type    | Description                      |
-| ------------ | ------- | -------------------------------- |
-| `isSimple`   | boolean | Whether it is without components |
-| `isInverted` | boolean | Whether it has inverted colors   |
-| `class`      | string  | Additional class name            |
+As the name suggests, desktop-only actions are intended to display on desktop
+screens only. There are two slots you can use: primary actions (aligned to
+left in LTR documents), and secondary actions (aligned to right).
```

Comment:
Writing commas in English text is still a problem for me, however should _Oxford comma_ be here? 

```suggestion
left in LTR documents) and secondary actions (aligned to right).
```